lib/screens/home_screen.dart

```diff
@@ -37,82 +46,32 @@ class _HomeScreenState extends State<HomeScreen> {
             ],
           ),
         ),
-        actions: [
-          IconButton(
-            icon: Icon(
-              TBIcons.edit,
-              color: TBColor.appBar.blueGradientColor,
-            ),
-            onPressed: () {},
-          )
-        ],
       ),
       body: Container(
-        child: SafeArea(
-          child: Center(
-            child: SingleChildScrollView(
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.end,
-                children: [
-                  TBProductCard(
-                    category: product.category,
-                    discount: product.discount,
-                    image: product.image,
-                    name: product.name,
-                    onStock: product.onStock,
-                    price: product.price,
-                    rating: product.rating,
-                    onTap: () {},
-                  ),
-                  TBSection(
-                      title: "Availability",
-                      content: TBAvailabilityCard(
-                        category: product.category,
-                        onStock: product.onStock,
-                      )),
-                  TBGallery.url(
-                    images: const [
-                      "https://upload.wikimedia.org/wikipedia/commons/b/b6/Image_created_with_a_mobile_phone.png",
-                      "https://upload.wikimedia.org/wikipedia/commons/a/a8/TEIDE.JPG",
-                    ],
-                  ),
-                  TBGallery.asset(
-                    images: const [
-                      "assets/iphone_image.png",
-                    ],
+        padding: EdgeInsets.symmetric(
+            horizontal: TBDimensions.homeScreen.contentPadding),
+        child: Center(
+          child: Column(
+            children: [
+              Expanded(
+                child: ListView.separated(
+                  padding: EdgeInsets.only(
+                    top: TBDimensions.homeScreen.contentPadding,
```

Comment:
List needs to have bottom padding too. Content should finish above floating button.